cf-check/cf-check.c

```diff
@@ -65,6 +68,11 @@ int main(int argc, char **argv)
     {
         return backup_main(cmd_argc, cmd_argv);
     }
+    if (StringSafeEqual_IgnoreCase(command, "repair")
+        || StringSafeEqual_IgnoreCase(command, "remediate"))
```

Comment:
why is remediate here but not documented?

Reply:
So that if @nickanderson  tries to do `cf-check remediate` it works ;) :P He seemed to prefer this over `repair` so I added both.